### ISSUE TITLE
Command line arguments

### ISSUE THREAD

**Issue by rivermont** [rivermont/spidy#46]
Arguments for:

 - [ ] Overwrite existing save files
 - [ ] Raise Errors (possibly only for different severity levels?)
 - [ ] Save pages
 - [ ] Save words
 - [ ] Zip files
 - [ ] Override file size check.
 - [ ] Domain restriction
 - [ ] Respect `robots.txt`
 - [ ] Custom save file locations
 - [ ] Autosave count
 - [ ] HTTP headers
 - [ ] Max errors
 - [ ] Starting page

*NOTES*: Don't try to use `sys.argv`.

**Comment by dasdachs** [comment]
If you are ok with (click)[http://click.pocoo.org/] I'd be happy to work on it.

**Comment by rivermont** [comment]
Sure! Just make sure to add click to `requirements.txt` and update `setup.py`.

**Comment by dasdachs** [comment]
Cool. I'll get at it as.

**Comment by mohamed-aziz** [comment]
It is a pain to work on this project, someone should fix it, adding this feature is pretty easy but because it has a lot of GLOBAL variables spattered all over the place it's not going to be that easy.

**Comment by rivermont** [comment]
@mohamed-aziz I agree, actually. This started as a simple script that I whipped in the spring, but it jumped to a full-blown project this past week. Would love some pointers on where to go from here!
